From c2ea2b38da6fd4f2dbfb1a26980edd3e3faf5876 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio at gentoo.org>
Date: Wed, 5 Dec 2018 18:06:53 +0200
Subject: [PATCH] build: Remove unnecessary gtk dependency for builder

Builder checks for gtk+, but the result isn't used anywhere, thus
introducing an unnecessary gtk+ dependency.
-Dbuilder=true -Dfonts=true would still depend on gdk, but a
-Dbuilder=true -Dfonts=false build doesn't anymore.

Closes: #274
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index a26b88a..a8649d5 100644
--- a/meson.build
+++ b/meson.build
@@ -60,7 +60,6 @@ gdkpixbuf = dependency('gdk-pixbuf-2.0', version : '>= 2.31.5')
 
 # builder (default enabled)
 if get_option('builder')
-  gtk = dependency('gtk+-3.0')
   gmodule = dependency('gmodule-2.0')
   if get_option('rpm')
     rpm = dependency('rpm')
-- 
2.18.1

